<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Body Mass Index Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>178</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Weight (kg)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLabel" name="text_bmi">
       <property name="text">
        <string>_bmi_</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinbox_weight">
       <property name="suffix">
        <string> kg</string>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSlider" name="slider_height">
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Gender</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="text_gender">
       <property name="text">
        <string>_gender_</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Height (cm)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinbox_height">
       <property name="suffix">
        <string> cm</string>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSlider" name="slider_weight">
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Body mass index</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLabel" name="text_interpretation">
       <property name="text">
        <string>_interpretation_</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Interpretation</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionFemale">
   <property name="text">
    <string>Female</string>
   </property>
  </action>
  <action name="actionMale">
   <property name="text">
    <string>Male</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>slider_weight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinbox_weight</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinbox_weight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_weight</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_height</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinbox_height</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinbox_height</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_height</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
